<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="Line" name="Seprator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>911</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="InputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="InputEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>441</width>
      <height>191</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter the text which you want to decrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecryptBtn">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>170</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
    <property name="autoRepeatDelay">
     <number>298</number>
    </property>
   </widget>
   <widget class="QLabel" name="OutputLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OUTPUT</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>240</y>
      <width>631</width>
      <height>171</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>This will give the decrypted text</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="KeyEdit">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>241</width>
      <height>181</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>eneter the ket it should not be one more then a multiple of 4</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="EncryptMenu">
    <property name="title">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KEncryptMenu">
    <property name="title">
     <string>Key Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KDecryptMenu">
    <property name="title">
     <string>Key Decrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="PasswordMenu">
    <property name="title">
     <string>Strong password</string>
    </property>
   </widget>
   <addaction name="EncryptMenu"/>
   <addaction name="KEncryptMenu"/>
   <addaction name="KDecryptMenu"/>
   <addaction name="PasswordMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
